app/src/main/java/com/kickstarter/libs/LakeEvent.kt

```diff
@@ -20,6 +20,7 @@ const val PLEDGE_SUBMIT_BUTTON_CLICKED = "Pledge Submit Button Clicked"
 const val PROJECT_PAGE_PLEDGE_BUTTON_CLICKED = "Project Page Pledge Button Clicked"
 const val SELECT_REWARD_BUTTON_CLICKED = "Select Reward Button Clicked"
 const val THANKS_PAGE_VIEWED = "Thanks Page Viewed"
+const val FIX_PLEDGE_BUTTON_CLICKED = "Fix Pledge Button Clicked"
```

Comment:
This event actually is in the `Manage a Pledge` set of events so we need a new region 🆕 
```
// region Manage a Pledge
const val FIX_PLEDGE_BUTTON_CLICKED = "Fix Pledge Button Clicked"
//endregion
```